tests/test_api.py

```diff
@@ -204,6 +204,17 @@ def test_index(app):
             assert fun.called
 
 
+def test_refresh(app):
+    """Test index refresh."""
+    with app.app_context():
+        client_mock = MagicMock()
+        ri = RecordIndexer(search_client=client_mock, version_type='force')
+        ri.refresh(app.config['INDEXER_DEFAULT_INDEX'])
```

Comment:
In this test we are passing a string. However, from `IndexRefreshOp` we are passing a `Index` object.